core/http/Cargo.toml

```diff
@@ -21,7 +21,9 @@ private-cookies = ["cookie/secure"]
 [dependencies]
 smallvec = "0.6"
 percent-encoding = "1"
-hyper = { version = "0.10.13", default-features = false }
+hyper = { version = "0.12.28", default-features = false, features = ["runtime"] }
```

Comment:
I chose to use `tokio` manually instead of via the `runtime` feature (for easier TLS handling, I think).

Reply:
Okay, I can change this as well.

Reply:
Feel free to change it in your branch, or not - I foresee a big project of a merge one of the coming weekends either way!

Reply:
@jebrosen One thing to keep in mind is that people will definitely want to run stuff other than Rocket on the Tokio event loop (e.g. async database calls using tokio-postgres). So either:

1. Rocket should (optionally) accept a tokio reactor to run on
2. Rocket should be able to expose it's own reactor for use by other code
3. The tokio runtime should be used (I *think* this allows for this).

Probably #1 with an option for #3 would be ideal?

Reply:
IIRC hyper 0.12 actually requires tokio runtime specifically (hyper 0.13 might not.) I definitely like the idea of allowing a user-configured runtime, though.